apps/api/webpack.config.js

```diff
@@ -1,13 +1,21 @@
 const { composePlugins, withNx } = require('@nx/webpack')
+const { sentryWebpackPlugin } = require('@sentry/webpack-plugin')
 
-// Nx plugins for webpack.
 module.exports = composePlugins(
   withNx({
-    target: 'node'
+    target: 'node',
+    devtool: 'source-map',
+    plugins: [
+      ...((process.env.NODE_ENV || 'production') === 'production'
```

Comment:
Think you can use this: if environment == `production` or `stage`, you can upload the sourcemaps